--- matplotlib-1.4.3/extern/qhull/qhull_a.h	2014-12-23 15:04:40.000000000 -0800
+++ patch-files/qhull_a.h	2015-10-08 18:03:48.287287537 -0700
@@ -103,10 +103,14 @@
 #   define QHULL_OS_WIN
 #endif
 #if defined(__INTEL_COMPILER) && !defined(QHULL_OS_WIN)
+#if 0
 template <typename T>
 inline void qhullUnused(T &x) { (void)x; }
 #  define QHULL_UNUSED(x) qhullUnused(x);
 #else
+#  define QHULL_UNUSED(x) (x);
+#endif
+#else
 #  define QHULL_UNUSED(x) (void)x;
 #endif
 
