--- matplotlib-1.4.2/setupext.py	2014-10-22 18:54:01.000000000 -0700
+++ patch-files/setupext.py	2014-10-26 18:15:54.232114435 -0700
@@ -298,7 +298,8 @@
             except subprocess.CalledProcessError:
                 pass
             else:
-                output = output.decode(sys.getfilesystemencoding())
+                #JJH: decode unicode result causes compilation errors
+                #output = output.decode(sys.getfilesystemencoding())
                 use_defaults = False
                 for token in output.split():
                     attr = flag_map.get(token[:2])
