--- matplotlib-1.4.3/setupext.py	2015-02-15 19:46:36.000000000 -0800
+++ patch-files/setupext.py	2015-10-08 17:59:34.852683477 -0700
@@ -297,7 +297,8 @@
             except subprocess.CalledProcessError:
                 pass
             else:
-                output = output.decode(sys.getfilesystemencoding())
+                #JJH: decode unicode result causes compilation errors
+                #output = output.decode(sys.getfilesystemencoding())
                 use_defaults = False
                 for token in output.split():
                     attr = flag_map.get(token[:2])
