--- setup.py	2015-01-27 11:39:09.140549177 -0800
+++ patch-files/setup.py	2015-01-27 11:40:16.433583504 -0800
@@ -47,6 +47,8 @@
     del sdist.sdist.make_release_tree
 
 from distutils.dist import Distribution
+# JJH: added to support intel
+from numpy.distutils import intelccompiler
 
 import setupext
 from setupext import print_line, print_raw, print_message, print_status
@@ -63,7 +65,8 @@
     setupext.Python(),
     setupext.Platform(),
     'Required dependencies and extensions',
-    setupext.Numpy(),
+    #JJH: Assume numpy installed w/python
+    #setupext.Numpy(),
     setupext.Six(),
     setupext.Dateutil(),
     setupext.Pytz(),
