--- numpy-1.8.0/site.cfg.example	2013-10-30 11:31:41.000000000 -0700
+++ patch-files/site.cfg.example	2014-03-14 15:26:26.802008615 -0700
@@ -59,8 +59,8 @@
 # This is a good place to add general library and include directories like
 # /usr/local/{lib,include}
 #
-#[DEFAULT]
-#library_dirs = /usr/local/lib
+[DEFAULT]
+library_dirs = /usr/lib64
 #include_dirs = /usr/local/include
 
 # Atlas
@@ -103,6 +103,12 @@
 # library_dirs = /opt/intel/mkl/10.0.1.014/lib/32/
 # lapack_libs = mkl_lapack
 # mkl_libs = mkl, guide
+ROLL_COMMENTMKL [mkl]
+ROLL_COMMENTMKL library_dirs = ROLL_MKLLIB
+ROLL_COMMENTMKL mkl_libs = mkl_rt
+ROLL_COMMENTMKL include_dirs =  ROLL_MKLROOT/include
+ROLL_COMMENTMKL lapack_libs =  mkl_rt
+
 
 # UMFPACK
 # -------
